subxt/src/backend/unstable/rpc_methods.rs

```diff
@@ -0,0 +1,890 @@
+            .transpose()?;
+        Ok(header)
+    }
+
+    /// Call the `chainhead_unstable_storage` method and return an operation ID to obtain the block's storage.
+    ///
+    /// The response events are provided on the `chainHead_follow` subscription and identified by
+    /// the returned operation ID.
+    ///
+    /// # Note
+    ///
+    /// The subscription ID is obtained from an open subscription created by
+    /// [`chainhead_unstable_follow`](UnstableRpcMethods::chainhead_unstable_follow).
+    pub async fn chainhead_unstable_storage(
+        &self,
+        subscription_id: &str,
+        hash: T::Hash,
+        items: impl IntoIterator<Item = StorageQuery<&[u8]>>,
+        child_key: Option<&[u8]>,
+    ) -> Result<MethodResponse, Error> {
+        let items: Vec<StorageQuery<String>> = items
+            .into_iter()
+            .map(|item| StorageQuery {
+                key: to_hex(item.key),
+                query_type: item.query_type,
+            })
+            .collect();
+
+        let response = self
+            .client
+            .request(
+                "chainHead_unstable_storage",
+                rpc_params![subscription_id, hash, items, child_key.map(to_hex)],
+            )
+            .await?;
+
+        Ok(response)
+    }
+
+    /// Call the `chainhead_unstable_storage` method and return an operation ID to obtain the runtime API result.
+    ///
+    /// The response events are provided on the `chainHead_follow` subscription and identified by
+    /// the returned operation ID.
+    ///
+    /// # Note
+    ///
+    /// The subscription ID is obtained from an open subscription created by
+    /// [`chainhead_unstable_follow`](UnstableRpcMethods::chainhead_unstable_follow).
+    pub async fn chainhead_unstable_call(
+        &self,
+        subscription_id: &str,
+        hash: T::Hash,
+        function: &str,
+        call_parameters: &[u8],
+    ) -> Result<MethodResponse, Error> {
+        let response = self
+            .client
+            .request(
+                "chainHead_unstable_call",
+                rpc_params![subscription_id, hash, function, to_hex(call_parameters)],
+            )
+            .await?;
+
+        Ok(response)
+    }
+
+    /// Unpin a block reported by the `chainHead_follow` subscription.
+    ///
+    /// # Note
+    ///
+    /// The subscription ID is obtained from an open subscription created by
+    /// [`chainhead_unstable_follow`](UnstableRpcMethods::chainhead_unstable_follow).
+    pub async fn chainhead_unstable_unpin(
+        &self,
+        subscription_id: &str,
+        hash: T::Hash,
+    ) -> Result<(), Error> {
+        self.client
+            .request(
+                "chainHead_unstable_unpin",
+                rpc_params![subscription_id, hash],
+            )
+            .await?;
+
+        Ok(())
+    }
+
+    /// Return the genesis hash.
+    pub async fn chainspec_v1_genesishash(&self) -> Result<T::Hash, Error> {
```

Comment:
```suggestion
    pub async fn chainspec_v1_genesis_hash(&self) -> Result<T::Hash, Error> {
```

Reply:
Maybe you're right; I went back and forth on this naming between "what feels nicest" and "what's more technically correct" :)

Reply:
(I did the renames :))